packages/web/src/components/Header/_Header.scss

```diff
@@ -0,0 +1,278 @@
+@use '../../tools/breakpoint';
+@use '../../tools/reset';
+@use '../../tools/svg';
+@use '../../tools/typography';
+@use 'theme';
+@use 'tools';
+
+.Header {
+    @include typography.generate(theme.$typography);
+
+    display: flex;
+    align-items: center;
+    flex: none;
+    gap: theme.$header-gap;
+    height: 68px;
+    padding-inline: theme.$header-padding-x;
+    background-color: transparent;
+}
+
+.Header__toggle {
+    margin-left: auto;
+}
+
+.Header__icon {
+    width: theme.$icon-size;
+    height: theme.$icon-size;
+    background-color: transparent;
+    background-repeat: no-repeat;
+}
+
+.Header__icon--close {
+    background-image: svg.escape(theme.$icon-close);
+}
+
+.Header__icon--menu {
+    background-image: svg.escape(theme.$icon-menu);
+}
+
+.Header__bar,
+.Header__dialog {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    display: grid;
+    width: 280px;
+    color: theme.$pane-color;
+    background-color: theme.$pane-background-color;
+    visibility: hidden;
+    z-index: 1;
+    transform: translateX(100%);
+
+    // Shadow and backdrop
+    &::before,
+    &::after {
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        visibility: hidden;
+    }
+
+    // Shadow
+    &::before {
+        right: 0;
+        left: 0;
+        box-shadow: theme.$pane-shadow;
+        z-index: -1;
+        opacity: 0;
+    }
+
+    // Backdrop
+    &::after {
+        right: 100%;
+        width: 100vw;
+        background-color: theme.$backdrop-background-color;
+        z-index: -2;
+        opacity: 0;
+    }
+
+    @media (prefers-reduced-motion: no-preference) {
+        transition-property: visibility, transform;
+        transition-timing-function: ease-in-out;
+        transition-duration: 0.25s;
+
+        // Shadow and backdrop
+        &::before,
+        &::after {
+            transition-property: opacity, visibility;
+            transition-timing-function: linear;
+            transition-duration: 0.25s;
+        }
+    }
+}
+
+.Header__content {
+    display: flex;
+    flex-direction: column;
+    overflow-y: auto;
+    -webkit-overflow-scrolling: touch;
+}
+
+.Header__actions {
+    display: flex;
+    flex-direction: column;
+    gap: theme.$actions-row-gap;
+    padding: theme.$actions-padding-y theme.$actions-padding-x;
+}
+
+.Header__actions--primary {
+    flex: none;
+}
+
+.Header__actions--secondary {
+    flex: 1;
+}
+
+.Header__nav {
+    margin-inline: -1 * theme.$actions-padding-x;
+    list-style: none;
+}
+
+.Header__navItem {
+    display: flex;
+}
+
+.Header__close,
+button.Header__link {
+    @include reset.button();
+}
+
+.Header__close {
```

Comment:
Maybe we should make the close button bigger?

https://uxmovement.com/mobile/optimal-size-and-spacing-for-mobile-buttons/ min 42px
https://www.kalamuna.com/blog/3-simple-tips-ux-button-design min 38px